app/models/user.rb

```diff
@@ -269,7 +269,14 @@ def self.from_omniauth(auth)
   def deliver_invitation(options = {})
     super(options.merge(subject: _('A Data Management Plan in %{application_name} has been shared with you') % {application_name: Rails.configuration.branding[:application][:name]}))
   end
-
+  ##
+  # Case insensitive search over User model
+  # @param field [string] The name of the field being queried
+  # @param val [string] The string to search for, case insensitive
+  # @return [ActiveRecord::Relation] The result of the search
+  def self.where_case_insensitive(field, val)
```

Comment:
In Rails 5 we can move this method to the new ApplicationRecord.rb to make it available to all of the models